presto-main/src/test/java/com/facebook/presto/block/AbstractTestBlock.java

```diff
@@ -146,7 +146,6 @@
 
             for (int offset = 0; offset < length - 3; offset++) {
                 assertEquals(block.getSlice(position, offset, 3), expectedSliceValue.slice(offset, 3));
-                assertEquals(block.hash(position, offset, 3), expectedSliceValue.hashCode(offset, 3));
```

Comment:
I don't think anyone relies on that and there is no reason to do this I think.
This is explicitly testing if our Block code is using exactly same hashing strategy as airlift Slice which doesn't make much sense, especially as airlift is using 32bit hash in this scenario.

I probably could replace expectedSliceValue.hashCode with XxHash64.hash(expectedSliceValue) to get equality, but does this really makes any sense to test that? It's like checking if we are still using same implementation.